Library/Homebrew/extend/os/linux/keg_relocate.rb

```diff
@@ -20,7 +20,9 @@ def change_rpath(file, old_prefix, new_prefix)
     # patchelf requires that the ELF file have a .dynstr section.
     # Skip ELF files that do not have a .dynstr section.
     return if ["cannot find section .dynstr", "strange: no string table"].include?(old_rpath)
-    raise ErrorDuringExecution, "#{cmd_rpath}\n#{old_rpath}" unless $CHILD_STATUS.success?
+    unless $CHILD_STATUS.success?
+      raise ErrorDuringExecution.new(cmd_rpath, status: $CHILD_STATUS, output: [:stdout, old_rpath])
```

Comment:
Is there a `safe` method this could use instead?

Reply:
There is a `return` between the call and the `raise`, so I don't think so.

Reply:
@reitermarkus Yeh, good point. I guess something could be `yield`ed but definitely overkill for one example.